packages/video_player/video_player/lib/video_player.dart

```diff
@@ -243,6 +244,27 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
         package = null,
         super(VideoPlayerValue(duration: Duration.zero));
 
+  /// Constructs a [VideoPlayerController] playing a video from obtained from
```

Comment:
Can you also remove the extra `from` from the comment you copied this from as well.